templates/workspace_services/azureml/user_resources/aml_compute/terraform/variables.tf

```diff
@@ -13,3 +13,4 @@ variable "arm_client_secret" {}
 variable "arm_use_msi" {
   type = bool
 }
+variable "tre_user_resources_tags" {}
```

Comment:
You don't need a _variable_ as you create tre_user_resources_tags as a _local_. Please remove this.